client/src/templates/stats.njk

```diff
@@ -0,0 +1,15 @@
+{% extends 'src/templates/root.njk' %}
+
+{% block head %}
+  <title>Stats</title>
+  <script src="/js/templates.js"></script>
+  <script src="https://mozilla.github.io/nunjucks/files/nunjucks-slim.min.js"></script>
```

Comment:
`nunjucks-slim` is much smaller but does not include compiling tools. That's ok because we are pre-compiling the client side templates.